source/prerequisites/postgres.txt

```diff
@@ -10,4 +10,78 @@ Postgres Migration Prerequisites
    :depth: 1
    :class: singlecol
 
-Postgres.
\ No newline at end of file
+About this Task
+---------------
+
+Connecting Relational Migrator to PostgreSQL requires
+`logical decoding 
+<https://www.postgresql.org/docs/current/logicaldecoding-explanation.html>`__ 
+enabled on your PostGreSQL instance. When you set up a new connection, 
+Relational Migrator automatically generates the required SQL code to 
+configure PostgreSQL. This document explains the purpose of those scripts.
+
+These instructions configure PostgreSQL **for both** Snapshot and 
+Continuous sync jobs.
+
+- Snapshot sync jobs migrate all the data and then stops.
+- Continuous sync job run a snapshot and then enter a CDC stage to 
+  continuously replicate data changes.
+
+About this Task
+---------------
+
+Relational Migrator uses the PostgreSQL ``pgoutput`` plugin for logical 
+decoding. This plug-in is included in PostgreSQL versions 10+, no 
+additional installation or configuration is required. 
+
+Before you Begin
+----------------
+
+If PostgreSQL is configured as a cluster, the tool must be connected 
+to a master server.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal
+
+
+   .. step:: Edit the Configuration file 
+
+      Logical replication may not be enabled by default. To enable logical 
+      replication change the `wal_level <https://postgresqlco.nf/doc/en/param/wal_level>`__ 
+      configuration in the 
+      `postgresql.conf configuration file <https://www.postgresql.org/docs/current/config-setting.html>`__. 
+      You **must restart the database instance** after changing the configuration file. 
+
+      .. code-block:: none
+
+         wal_level = logical 
+
+   .. step:: Create a SQL Replication Login
+
+      The PostgreSQL account requires ``REPLICATION`` and ``LOGIN`` 
+      database permissions as well as ``SELECT`` and ``USAGE`` 
+      permissions on each source table:
+
+      .. code-block:: sql
```

Comment:
For Tech Review:

Should we add example grant statements for the permissions mentioned above?